RadixWallet/Profile/Factor/FactorSourceKind/FactorSourceKindOfMnemonic.swift

```diff
@@ -1,7 +1,7 @@
 
 public enum FactorSourceKindOfMnemonic: Sendable, Hashable {
 	public enum OnDeviceMnemonicKind: Sendable, Hashable {
-		case babylon
+		case babylon(isMain: Bool)
```

Comment:
hmm, does this require profile migration?

Reply:
no this is not anything saved in Profile. It is NOT the `FactorSourceKind`.... it is just a helper DTO.. (not `Codable`)